django/views/generic/edit.py

```diff
@@ -233,6 +239,18 @@ class BaseDeleteView(DeletionMixin, FormMixin, BaseDetailView):
     """
     form_class = Form
 
+    def __init__(self, *args, **kwargs):
+        # RemovedInDjango50Warning.
```

Comment:
What if a view based on `DeleteView` overrides `delete` method to actually handle requests with HTTP `DELETE` method? For them this warning would be a false positive.

Reply:
I'm not sure what else could we do :thinking: maybe move this check to the `post()` :shrug:

Reply:
I prefer to have it here, because this way the warning appears on startup.
If we do it to `post()`, then the warning will appear only in runtime - when handling request. 
Better to keep it here and assume that `DELETE` http method is an edge case for `DeleteView` :)

P.S. It would be helpful to include actual view class name to the warning message, to make it obvious which view must be updated.

Reply:
> P.S. It would be helpful to include actual view class name to the warning message, to make it obvious which view must be updated.

Added.